docs/configuration-reference/components/aws-ebs-csi-driver.md

```diff
@@ -0,0 +1,47 @@
+# CSI Driver for Amazon EBS configuration reference for Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Configuration](#configuration)
+* [Applying](#applying)
+* [Destroying](#destroying)
```

Comment:
After https://github.com/kinvolk/lokomotive/pull/481, it should be `Deleting`, so we stay consistent.